### PR TITLE
Re-add old LBP UI

### PR DESCRIPTION
**Pull request by jonator**
Add back the LBP UI from the pre-refactor frontend so that users can view the moving weights of any active and configured LBP pool.

Note/disclaimer: this is not a specific effort or addition for the Rebus chain. We are not affiliated and our frontend and chain remains permission-less.

### CODE REVIEW

File: packages/web/pages/_app.tsx [hunk 1]
```diff
@@ -28,7 +29,7 @@ dayjs.extend(utc);
 enableStaticRendering(typeof window === "undefined");
 
 function MyApp({ Component, pageProps }: AppProps) {
-  const menus = [
+  let menus: MainLayoutMenu[] = [
```
Comment (delivan) [inline packages/web/pages/_app.tsx:32]: It seems that It don't need to change declare type.
```suggestion
  const menus: MainLayoutMenu[] = [
```

**Comment by delivan** [comment]
@jonator It seems that it is also necessary to disable to access pool detail page and swap tokens related LBP.

**Comment by dogemos** [comment]
also can we make sure this only gets enabled via frontier and not through app.osmosis.zone? or would this require a ton of work?

**Comment by jonator** [comment]
> also can we make sure this only gets enabled via frontier and not through app.osmosis.zone? or would this require a ton of work?

Trivial work. And it's added.

**Comment by delivan** [comment]
@jonator I have something misunderstood. 😅

**Comment by jonator** [comment]
@delivan what do you mean?

**Comment by RowanUtrecht** [comment]
@jonator 
It looks like the price isn't correct. Since the swap ratio between Osmo and Rebus is going down. But the weight of the pool stays at about 96/4. That's not correct

Screenshot_20220930-080448_Brave
Screenshot_20220930-080439_Brave

**Comment by RowanUtrecht** [comment]
Sorry, checked again. Price looks good. The LBP is just very slowly moving to the 50/50 ratio. Probably some people keep buying small amounts

**Pull request merged**